spec/features/feedback_spec.rb

```diff
@@ -13,6 +13,8 @@
     feedback_page.fill_in_feedback(feedback_factory)
     feedback_page.submit
 
+    expect(page).to have_text "Thank you for your feedback"
```

Comment:
You have written the capybara selector but not used it ? (which means this wont work in welsh)

Maybe you could do something like

```ruby
expect(feedback_page).to have_thank_you_message
```

Then inside the feedback_page object

```
element :thank_you_message, :flash_message_with_text('your.path.to.the.thank.you.message.in.locale.file')
```

and add translation into the flash_message_with_text capybara selector the same as the others ?